tools/find-unreleased-projects.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I'm on a Mac, so... :)

```suggestion
#!/usr/bin/env bash
```

Reply:
d'oh!